src/parser/flex_lexer.l

```diff
@@ -186,18 +186,25 @@ OF			TOKEN(OF)
 ON			TOKEN(ON)
 OR			TOKEN(OR)
 TO			TOKEN(TO)
-SECOND      TOKEN(SECOND)
-MINUTE      TOKEN(MINUTE)
-HOUR        TOKEN(HOUR)
-DAY         TOKEN(DAY)
-MONTH       TOKEN(MONTH)
-YEAR        TOKEN(YEAR)
+SECOND		TOKEN(SECOND)
+MINUTE		TOKEN(MINUTE)
+HOUR		TOKEN(HOUR)
+DAY			TOKEN(DAY)
```

Comment:
The intendation in `bison_parser.y`  and `flexer_lexer.l` is kind of messy at all (mostly tabs, but sometimes four spaces). Indeed, tabs here (and for some of the tokens, spaces and tabs are mixed) look kind of weird when printed to console, but when they are fixed to contain tabs only (e.g., with nano), the intendation is not correct in a text editor (e.g., Sublime) anymore.
I would have no problem to fix this forever by applying a space intendation everywhere.

Reply:
For reference, see the same lines of `flexer_lexer.l` opened with nano and Sublime:
![image](https://user-images.githubusercontent.com/27929897/133651479-5e40f3a6-7aa6-4627-857a-fc27750e0c31.png)
![image](https://user-images.githubusercontent.com/27929897/133651503-4a03b68b-7e34-42bd-894b-c31aaf887569.png)

Reply:
Having space indentation consistently would be the way to go in my opinion. However, the indentation you used in your added code looks consistent (tab indentation). For this PR, fixing the indentation issue is not required imo since you did not introduce it with this PR. However, feel free to fix it, if you want to. :)

Reply:
I think openening a new PR for this once this one is merged is the cleanest solution.